File: Ejercicios/Java/Ejercicio3.java
import java.util.ArrayList;
import java.util.Scanner;

public class Ejercicio3 {
    public static void main(String[] args) {
        ArrayList<String> prods = new ArrayList<>();

        prods.add("Teclado");
        prods.add("Mouse");
        prods.add("Monitor");

        Scanner sc = new Scanner(System.in);

        System.out.print("Buscar producto: ");
        String buscar = sc.next();

        if (prods.contains(buscar)) {
            System.out.println("Producto encontrado");
        } else {
            System.out.println("No existe");
        }

        sc.close(); 
    }
}

File: Ejercicios/C++/Ejercicio3.cpp
#include <iostream>
#include <vector>
using namespace std;
int main() {
    vector<string> prods; 
    prods.push_back("Teclado"); prods.push_back("Mouse"); prods.push_back("Monitor");
    string buscar; cout << "Buscar producto: "; cin >> buscar;
    bool enc = false;
    for(string p : prods) if(p == buscar) enc = true;
    cout << (enc ? "Producto encontrado\n" : "No existe\n");
    return 0;
}
